tests/default/_core/search/geo_bounding_box.yaml

```diff
@@ -0,0 +1,61 @@
+$schema: ../../../../json_schemas/test_story.schema.yaml
+
+description: Test search endpoint with geo_bounding_box query.
+prologues:
+  - path: /point_index
```

Comment:
Rename `point_index` to something more in the themes we have of movies and books, maybe `cinemas` with fields such as `location` (rather than `field` which hides bugs)? Since the plan is to auto-generate docs it will be nice to have examples that are consistent.